drv/vsc7448/src/spi.rs

```diff
@@ -0,0 +1,276 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::VscError;
+use drv_spi_api::SpiDevice;
+use ringbuf::*;
+use vsc7448_pac::{types::RegisterAddress, Vsc7448};
+
+#[derive(Copy, Clone, PartialEq)]
+enum Trace {
+    None,
+    Read { addr: u32, value: u32 },
+    Write { addr: u32, value: u32 },
+}
+
+/// This indicates how many bytes we pad between (writing) the address bytes
+/// and (reading) data back, during SPI transactions to the VSC7448.  See
+/// section 5.5.2 for details.  1 padding byte should be good up to 6.5 MHz
+/// SPI clock.
+pub const SPI_PAD_BYTES: u8 = 1;
```

Comment:
You seem to use this const as a usize -- I suggest making it a usize. That would prevent misinterpretation that 1 _is the pad byte._

Reply:
Good idea; I also renamed it to `SPI_NUM_PAD_BYTES` for clarity.